include/cantera/oneD/Flow1D.h

```diff
@@ -670,41 +676,242 @@ class Flow1D : public Domain1D
 
     //! @name Convective spatial derivatives
     //!
-    //! These use upwind differencing, assuming u(z) is negative
+    //! These methods use upwind differencing to calculate spatial derivatives
+    //! for velocity, species mass fractions, and temperature. Upwind differencing
+    //! is a numerical discretization method that considers the direction of the
+    //! flow to improve stability.
     //! @{
+
+    /**
+     * This function calculates the spatial derivative of velocity V with respect
+     * to z at point j using upwind differencing.
+     *
+     * @f[
+     *   \frac{\partial V}{\partial z} \bigg|_{j} \approx \frac{V(x, j_{\text{loc}}) -
+     *   V(x, j_{\text{loc}} - 1)}{m_{\text{dz}}[j_{\text{loc}} - 1]}
+     * @f]
+     *
+     * Where the value of loc is determined by the sign of the axial velocity.
+     * If the axial velocity is positive, the value of loc is j. If the axial velocity
+     * is negative, the value of loc is j + 1. A positive velocity means that the flow
+     * is moving left-to-right.
+     *
+     * @param[in] x The local domain state vector.
+     * @param[in] j The index at which the derivative is computed.
+     */
     double dVdz(const double* x, size_t j) const {
-        size_t jloc = (u(x,j) > 0.0 ? j : j + 1);
-        return (V(x,jloc) - V(x,jloc-1))/m_dz[jloc-1];
+        size_t jloc = (u(x, j) > 0.0 ? j : j + 1);
+        return (V(x, jloc) - V(x, jloc-1))/m_dz[jloc-1];
     }
 
+    /**
+     * This function calculates the spatial derivative of the species mass fraction
+     * @f$ Y_k @f$ with respect to z for species k at point j using upwind
+     * differencing.
```

Comment:
This is a better example where my previous comment applies.

Reply:
One (minor) step to keeping docstrings succinct is to omit the obvious subject, for example "This function" and just start with "Calculate the spatial derivative...".

Reply:
Comment wasn’t sufficiently addressed.

Reply:
Ah, good catch. I could have swore I got all of those instances of redundant comments.